feature-libs/quote/components/actions/link/quote-actions-link.component.html

```diff
@@ -2,7 +2,7 @@
   <section>
     <ul>
       <li>
-        <a (click)="goToNewCart()" class="link cx-action-link">
+        <a tabindex="0" (click)="goToNewCart()" class="link cx-action-link">
```

Comment:
There is `add-to-saved-cart.component` that behaves the same way as this one, namely there are 2 links that enables the navigation accordingly.
Why not to add `cxAutoFocus` to both links and ` tabindex="0"` to the 2. one?
I would even go further and suggest to refactor this component that its structure looks the same as `add-to-saved-cart.component`

Reply:
@Larisa-Staroverova: why shall we add `cxAutoFocus `directive - we only want both links to be in the tab chain, we do not want to change the focus behavior.

Reply:
@Uli-Tiger, good point!
@steinsebastian, what do you think where the focus should be set when one navigates to the quote overview, for instance from the quote list?

Reply:
ok i'll merge now and lets take the focus discussion offline.